templates/includes/card.html

```diff
@@ -15,17 +15,19 @@
 -->
 
 {% from './pill.html' import pills %} {% macro card(api, type) %}
-<li id="{{api.title | slug}}" class="card" data-title="{{api.title}}" data-shipping="{{api.shipping | dump}}" data-type="{{type}}" data-bug="{{api.id}}" data-where="{{api.where | dump | lower}}" data-pwa="{{api.pwa}}">
+<li id="{{api.title | slug}}" class="card" data-title="{{api.title}}" data-shipping="{{api.shipping | dump}}" data-type="{{type}}" data-bug="{{api.id}}" data-where="{{api.where | dump | lower}}" data-pwa="{{api.pwa}}" data-iwa="{{api.iwa}}">
```

Comment:
I think* the "data-iwa" is used for search. Did you add a "Isolated Web App" filter?

<img width="281" alt="image" src="https://user-images.githubusercontent.com/634478/192222557-d7d4ff4b-0709-49fd-9bd1-b3fee151c182.png">

Reply:
The filter is there, yes!